### PR TITLE
fix(/contribute/page-templates): broken redirect

### LINKED ISSUE #27917: /contribute/page-templates returns 404
**This is a Bug Report**

**Problem:** "Not found" / 404 returned by https://kubernetes.io/docs/contribute/style/page-templates/

**Proposed Solution:**
* Fix redirect (need to identify correct page)
* Seems other pages just redirect to [/docs/contribute/](https://kubernetes.io/docs/contribute) - could that be a solution?

**Page to Update:**
* https://kubernetes.io/docs/contribute/style/page-templates/

### PR DESCRIPTION
**Pull request by jai**
Fixes #27917 

Signed-off-by: Jai Govindani <jai@honestbank.com>

### CODE REVIEW

**Comment by jai** [comment]
Confirmed that a previously 404: https://deploy-preview-28107--kubernetes-io-master-staging.netlify.app/docs/contribute/style/page-templates/

redirects to: https://deploy-preview-28107--kubernetes-io-master-staging.netlify.app/docs/contribute/

**Comment by jai** [comment]
@jimangel hi PR wrangler 👋🏽

File: static/_redirects [hunk 1]
```diff
@@ -189,6 +189,7 @@
 /docs/home/contribute/review-issues/ /docs/contribute/intermediate/ 301
 /docs/home/contribute/blog-post/ /docs/contribute/start/ 301
 /docs/home/contribute/write-new-topic/ /docs/contribute/style/write-new-topic/ 301
+/docs/contribute/style/page-templates/ /docs/contribute/ 301
```
Comment (kbhawkey) [inline static/_redirects:192]: Hi @jai .
I looked back to v1.17. If I recall correctly, the template page, ` /docs/contribute/style/page-templates/ `, was replaced by the content types page,  `/docs/contribute/style/page-content-types/`.

What do you think about updating line 187 to point to the `page-content types` page or to the main style page,
`/docs/contribute/style/`?

Line 192 could also redirect to `/docs/contribute/style` or `/docs/contribute/style/page-content-types`.

Comment (jai) [reply static/_redirects:192]: @kbhawkey sure I'll push the commits soon, thanks for the comment!

Comment (jai) [reply static/_redirects]: done!

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-28107--kubernetes-io-master-staging.netlify.app/docs/contribute/style/page-templates/

**Comment by kbhawkey** [comment]
Thanks @jai .
/approve

File: static/_redirects [hunk 2]
```diff
@@ -184,11 +184,12 @@
 /docs/home/contribute/generated-reference/kubernetes-api/ /docs/contribute/generate-ref-docs/kubernetes-api/ 301
 /docs/home/contribute/generated-reference/kubernetes-components/ /docs/contribute/generate-ref-docs/kubernetes-components/ 301
 /docs/home/contribute/localization/ /docs/contribute/localization/ 301
-/docs/home/contribute/page-templates/ /docs/contribute/style/page-templates/ 301
-/docs/home/contribute/participating/ /docs/contribute/participate/ 301
```
Comment (sftim) [inline static/_redirects:188]: This is correct I think.
https://kubernetes.io/docs/contribute/participate/ looks like a page we should redirect to.

**Review by sftim — CHANGES_REQUESTED** [review]
Hi @jai

I think these changes aren't quite right, I'm afraid. Would you like help revising them further?

File: static/_redirects (hunk 2 above)
Comment (jai) [reply static/_redirects:188]: fixed

**Comment by jai** [comment]
@sftim should be good now

**Comment by kbhawkey** [comment]
Thanks @jai . Looks better.
/label tide/merge-method-squash

**Review by sftim — COMMENTED** [review]
/lgtm

**Pull request merged**